ephios/core/templates/core/disposition/fragment_participation.html

```diff
@@ -57,13 +60,26 @@ <h6 class="d-inline">
             <div class="card-body">
                 {{ form.individual_start_time|as_crispy_field }}
                 {{ form.individual_end_time|as_crispy_field }}
-                {% if form.comment.initial %}
-                    {{ form.comment|as_crispy_field }}
+                {% if form.previous_comments %}
+                    {{ form.previous_comments|as_crispy_field }}
                 {% endif %}
+                <div class="row">
+                    <div class="form-group col">
+                        {{ form.comment|as_crispy_field }}
+                    </div>
+                    <div class="form-group col-md-2 pt-4">
+                        <input type="checkbox" class="btn-check" id="{{ form.comment_is_internal.auto_id }}" name="{{ form.comment_is_internal.html_name }}" autocomplete="off">
+                        <label class="btn btn-outline-primary" for="{{ form.comment_is_internal.auto_id }}"><span class="fa fa-eye-slash"></span></label>
+                    </div>
+{#                    <div class="form-group col">#}
```

Comment:
not happy with this, but out of ideas on how to display it

Reply:
![image](https://github.com/user-attachments/assets/f42aebec-3079-4622-932d-a56defaa6e7a)
I agree this does not explain itself. why not a checkbox "show to participant"? Is that even what this does?